scripts/es-integration-test.sh

```diff
@@ -58,6 +58,23 @@ setup_query() {
   SPAN_STORAGE_TYPE=${distro} ./cmd/query/query-${os}-${arch} ${params[@]}
 }
 
+wait_for_it() {
+  local url=$1
+  local params=(
+    --silent
+    --output
+    /dev/null
+    --write-out
+    ''%{http_code}''
+  )
+  local counter=0
+  while [[ "$(curl ${params[@]} ${url})" != "200" && ${counter} < 30 ]]; do
+    sleep 2
+    counter=$((counter+1))
+    echo "waiting for ${url} to be up..."
+  done
+}
```

Comment:
I would recommend failing the whole script if 200 was never achieved, this way we'd get a much earlier and clearer failure signal.

Reply:
i.e. something like this:
```
while counter < 30 {
    status = $(curl)
    if status == 200 then return 0
}
return 1
```